src/enforcer.js

```diff
@@ -158,6 +124,73 @@ export class TrustedTypesEnforcer {
      * @private {Object<string, function(*): *|undefined>}
      */
     this.originalSetters_ = {};
+    /**
+     * The object that will be monkey patched by the polyfill.
+     * @private {Window}
+     */
+    this.windowObject = windowObject ||
+      (typeof window !== 'undefined' ? window : null);
+
+    if (!this.windowObject || typeof this.windowObject !== 'object') {
+      throw new Error(
+          // eslint-disable-next-line
+          'The polyfill expects a global `window` object or emulated `window-like` object passed to the enforcer as second argument'
+      );
+    }
+
+    // In IE 11, insertAdjacent(HTML|Text) is on HTMLElement prototype
+    this.insertAdjacentObjectPrototype = ((w) => {
+      if (!w.Element || !w.HTMLElement) return null;
+      return apply(hasOwnProperty, w.Element.prototype,
+          ['insertAdjacentHTML']) ?
+          w.Element.prototype :
+            w.HTMLElement.prototype;
+    })(this.windowObject);
+  }
+
+  /**
+     * Checks whether the value is instanceOf the specific window object.
+     * @param {*} value
+     * @param {!string} winProp
+     * @return {!boolean}
+     * @private
+     */
+  instanceOfDomProperty(value, winProp) {
```

Comment:
The convention for private fields and methods is to suffix them with _, still (unlike in TS). Please change here and in e.g. windowObject.

Reply:
Done. (That is a wierd convention though...)